runtime.go

```diff
@@ -578,9 +578,18 @@ func ReleaseBuffer(b *bytes.Buffer) {
 	bufferPool.Put(b)
 }
 
+type stringable interface {
+	~int | ~int8 | ~int16 | ~int32 | ~int64 |
+		~uint | ~uint8 | ~uint16 | ~uint32 | ~uint64 | ~uintptr |
+		~float32 | ~float64 |
+		~string |
+		~bool |
+		~complex64 | ~complex128
+}
```

Comment:
The formatting looks off.

Reply:
I was sort of surprised by the formatting too, I tried to format by kind of primitive, then gofmt did the indentation. I'll try some alternatives.